loader/include/Geode/utils/web.hpp

```diff
@@ -122,12 +134,20 @@ namespace geode::utils::web {
 
         WebRequest& userAgent(std::string_view name);
 
+        WebRequest& encoding(std::string_view encoding);
+
         WebRequest& timeout(std::chrono::seconds time);
 
+        WebRequest& range(std::pair<std::uint64_t, std::uint64_t> byteRange);

```

Comment:
what is this for

Reply:
the range of bytes to download from a response